### ISSUE TITLE
Bundler/InsecureProtocolSource shoudl detect http://rubygems.org

### ISSUE THREAD

**Issue by tas50** [rubocop/rubocop#10103]
## Is your feature request related to a problem? Please describe.

The `Bundler/InsecureProtocolSource` cop should detect http://rubygems.org and replace it with the https version.
## Additional context

This should error:

```ruby
source 'http://rubygems.org'

gem 'coveralls', require: false
```
